testing/tests/DevExpress.core/config.tests.js

```diff
@@ -88,3 +88,31 @@ QUnit.test('deprecated fields', function(assert) {
         errors.log = originalLog;
     }
 });
+
+QUnit.test('DevExpress.config.optionsParser works correct', function(assert) {
+    const optionsParser = DevExpress.config().optionsParser;
+
+    const testData = [
+        [
+            'dxTemplate: { "name": "title" }',
+            { 'dxTemplate': { 'name': 'title' } },
+        ],
+        [
+            // eslint-disable-next-line quotes
+            "dxTemplate: { name: 'title', test: { arr: ['a1', 'a2'], }, }",
+            { 'dxTemplate': { 'name': 'title', test: { arr: ['a1', 'a2'] } } }
+        ],
+        [
+            '"dxTemplate": { "name": "title" }',
+            { 'dxTemplate': { 'name': 'title' } }
+        ],
+        [
+            '{"dxTemplate": { "name": "title" }}',
+            { 'dxTemplate': { 'name': 'title' } }
+        ]
```

Comment:
We check the `\n` symbol in regular expression, but there is no test with multiline template. Maybe add it?

Reply:
fixed